### PR TITLE
Convert async task to sync

### PR DESCRIPTION
**Pull request by betoredhat**
The jobid files are not been cleaned up from the ansible workstation

### CODE REVIEW

**Comment by betoredhat** [comment]
Confirmed that the the temp file is removed in https://www.distributed-ci.io/jobs/1192db49-c671-457d-a633-39ec159e2a40/jobStates?sort=date&task=075ebb0e-cfdb-4bb0-8986-fe2e2b110816

File: roles/example-cnf-app/tasks/app.yaml [hunk 1]
```diff
@@ -11,6 +11,20 @@
   poll: 2
   failed_when: "net_list|length == 0"
 
+- name: Wait for net-attach-def presence
```
[the remark this answers is not in the document]
Comment (betoredhat) [reply roles/example-cnf-app/tasks/app.yaml]: yeah, hehe... I was trying to fix the problem with the directory files left behind... but not sure why async was required.

Updated the check

Thanks.

**Comment by betoredhat** [comment]
Tested in https://www.distributed-ci.io/jobs/73b6bb58-e6df-430d-9ffb-5deca4c4aff1/jobStates?sort=date&task=0beeb5de-aced-4c7b-8ea7-0b51a80be5e4

**Review by tonyskapunk — APPROVED** [review]
:+1:

**Pull request merged**